### DISCUSSION TITLE
multi-step, wizard component

### DISCUSSION THREAD

**Discussion by rezgui** [orchidsoftware/platform#2125] [Ideas]
multi-step, wizard component for sequential data collection.

**Comment by tabuna** [comment]
Hi @rezgui. You can use sessions and middleware to store intermediate results and determine which steps to display.
Roughly it would look like this:

```php
Route::middleware([Wizzard::class])->group(function () {
    Route::screen('/step/1', OneStepScreen::class);
    Route::screen('/step/2', TwoStepScreen::class);
});
```

**Reply by rezgui** [reply]
Thanks @tabuna , i will try this now

**Reply by rezgui** [reply]
Just for info, now i try make a New component step/wizard for orchid. Your msg down in Time. Thanks
